tests/otel_tracing_e2e/test_e2e.py

```diff
@@ -120,24 +119,42 @@ class Test_OTelLogE2E:
     def setup_main(self):
         self.r = weblog.get(path="/basic/log")
         self.use_128_bits_trace_id = False
+        time.sleep(5)  # wait a bit for logs agent to submit logs
```

Comment:
You must achieve this by adding a argument to this line : 

https://github.com/DataDog/system-tests/blob/d0e37b5accdc81975e22564d83a23e46e2a3411e/utils/_context/_scenarios.py#L1102

The argument is `backend_interface_timeout`, it's 0 by default, so I guess you want at least 20. You must also remove any sleep in `setup` methods (sorry, I miss that point in a preview PR review ...)

Reply:
Makes sense, used it instead